src/components/PackPage/SoloPostView/SoloPlaydates.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect, useContext, useRef } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import SoloPackName from './SoloPackName.jsx';
+import axios from 'axios';
+import SoloPlaydate from './SoloPlaydate.jsx';
+
+const SoloPlaydates = ({ setViewing, userIdentity }) => {
+  // var listNames = ['Woofram Alpha', 'Barkalona', 'Bark Simpson'];
+
+  var [playdates, setPlaydates] = useState([]);
+
+  useEffect(() => {
+    axios
```

Comment:
verify that you do not need to set the baseURL for axios here? I saw you did it elsewhere, not sure if it got passed down here or if you need to explicitly define it